cmd/otelcontribcol/Dockerfile

```diff
@@ -5,9 +5,11 @@ FROM scratch
 
 ARG USER_UID=10001
 USER ${USER_UID}
+# Args automaticaly filled by buildkit
+ARG TARGETOS TARGETARCH
```

Comment:
Changing `buildx build --platform linux/amd64,linux/arm64` will automaticaly change these args, so we can add new platforms tomorrow if we build binaries for other platorms, just changing platforms targeted in the build command.